core/themes/bartik/color/color.inc

```diff
@@ -15,14 +15,16 @@ $info = array(
     'titleslogan' => t('Title and slogan'),
     'text' => t('Text color'),
     'link' => t('Link color'),
+    'menu' => 'Menu link',
+    'activemenu' => t('Active menu link color'),
   ),
   // Pre-defined color schemes.
   'schemes' => array(
     'default' => array(
-      'title' => t('Blue Lagoon (default)'),
+      'title' => t('Blue Drop (default)'),
```

Comment:
For this PR the default color is Blue Drop. Are we changing to black? We decided to keep Blue Lagoon (the old Bartik gradient) for legacy reasons, so I chose "Blue Drop" (flat blue) to distinguish the two blues.

If we have decided on black as default, I think "Black Drop" sounds nice (a play on Back Drop) and although it doesn't jive with your minimalist preference I think a more quirky title matches the one its replacing (Blue lagoon), and the other schemes, none of which are colors, but schemes (Mocha, Giant Goldfish, Ice).